src/router/index.js

```diff
@@ -92,13 +92,21 @@ export const constantRoutes = [
     path: '/documentation',
     component: Layout,
     redirect: '/documentation/index',
-    meta: { title: 'documentation', icon: 'documentation', affix: true, breadcrumb: false },
+    meta: {
```

Comment:
I don't know what is the reason for it

Reply:
the affix attribute controls whether it is allowed to be closed while false or not like the dashboard.